rust_src/src/eval_macros.rs

```diff
@@ -283,3 +283,26 @@ macro_rules! per_buffer_var_idx {
         (unsafe { crate::remacs_sys::buffer_local_flags.$field }).as_natnum_or_error() as usize
     };
 }
+
+// Creates a Lisp_String from $string on the stack and a LispObject
+// pointing the Lisp_String. Assigns the LispObject to $name
+#[macro_export]
+#[allow(unused_macros)]
+macro_rules! local_unibyte_string {
+    ($name: ident, $string: expr) => {
```

Comment:
Should the code be placed in a block to ensure the names do not conflict? For instance, what happens if you try to call this macro twice in the same namespace?

Reply:
Run a test where you call the macro twice in a row and see if the output is right.

Reply:
Added the test `test_local_unibyte_string`.